app/javascript/tests/layout/Header.test.js

```diff
@@ -3,27 +3,33 @@ import { shallow } from 'enzyme';
 import Header from '../../components/layout/Header';
 import { mockUser1 } from '../mocks/mockUsers';
 
-let help_links = {
+const helpLinks = {
   user_guides: 'https://www.sara.org/user-guides',
   user_forum: 'https://www.sara.org/user-forum',
   contact_us: 'https://www.sara.org/contact-us',
 };
 
+/**
+ * If additionalProps undefiend, the Header component is rendered with the default props included below
+ * If any props are defined in additional props, they overwrite whatever the default value was set as
+ */
+function getWrapper(additionalProps) {
+  return shallow(<Header report_mode={false} show_demo_warning_background={false} banner_message="" current_user={mockUser1} help_links={helpLinks} {...additionalProps} />);
+}
+
 describe('Header', () => {
   it('Shows help link menu with all help links', () => {
-    const wrapper = shallow(<Header report_mode={false} show_demo_warning_background={false} banner_message="" current_user={mockUser1} help_links={help_links} />);
+    const wrapper = getWrapper();
 
     expect(wrapper.find('.dropdown-menu').exists()).toBe(true);
     expect(wrapper.find('.dropdown-menu').children()).toHaveLength(3);
-    expect(wrapper.find('.dropdown-item').find('a').at(0).prop('href')).toEqual('https://www.sara.org/user-guides');
-    expect(wrapper.find('.dropdown-item').find('a').at(1).prop('href')).toEqual('https://www.sara.org/user-forum');
-    expect(wrapper.find('.dropdown-item').find('a').at(2).prop('href')).toEqual('https://www.sara.org/contact-us');
+    expect(wrapper.find('.dropdown-item').find('a').at(0).prop('href')).toEqual(helpLinks.user_guides);
+    expect(wrapper.find('.dropdown-item').find('a').at(1).prop('href')).toEqual(helpLinks.user_forum);
+    expect(wrapper.find('.dropdown-item').find('a').at(2).prop('href')).toEqual(helpLinks.contact_us);
   });
 
   it('Shows help link menu with only two help links', () => {
-    help_links.user_guides = null;
-
-    const wrapper = shallow(<Header report_mode={false} show_demo_warning_background={false} banner_message="" current_user={mockUser1} help_links={help_links} />);
+    const wrapper = getWrapper({ help_links: { user_guides: null, user_forum: 'https://www.sara.org/user-forum', contact_us: 'https://www.sara.org/contact-us' } });
```

Comment:
My bad for not saying this in my original comment, but you can do `user_forum: helpLinks.user_forum` then check for that value a couple lines below, that way it's not hard coded

Reply:
also this is a nitpick but the newlines within the tests aren't necessary since the tests are so short and easy to read

Reply:
Sure! 

[339cb0e](https://github.com/SaraAlert/SaraAlert/pull/1243/commits/339cb0e798299df1b1960d7c77473534e478e2e7)